src/nvim/eval.c

```diff
@@ -400,6 +400,8 @@ static struct vimvar {
   VV(VV_SEARCHFORWARD,  "searchforward",    VAR_NUMBER, 0),
   VV(VV_HLSEARCH,       "hlsearch",         VAR_NUMBER, 0),
   VV(VV_OLDFILES,       "oldfiles",         VAR_LIST, 0),
+  VV(VV_ACTUAL_CURBUF,  "actual_curbuf",    VAR_NUMBER, VV_RO),
+  VV(VV_STL_WINDOW,     "stl_window",       VAR_NUMBER, VV_RO),
```

Comment:
Probably `v:actual_curwin` is a better choice. It's not really statusline-specific, semantically, though that happens to be the only place it's useful. And even if existing names are ugly, it's better to follow their pattern than add a new pattern.

Also agreed with @tjdevries : we need to keep `g:actual_curbuf`, but in the documentation move it to `deprecated.txt`. And introduce a redundant `v:` variant, `v:actual_curbuf`.

Reply:
But this is not at all `curwin`, rather "the window for which the statusline is currently drawn". It is the logical opposite of `actual_curbuf`.

Reply:
Ah, I see. Hmm.

Reply:
If you mean new var, why not `v:stl_buffer` ? If you mean `v:actual_curbuf` I see no point of renaming it.

Reply:
What i mean is if the two variables `v:actual_curbuf` and `v:stl_window` are analogous they should be named analogously.

Reply:
Alright, I should not have commented without looking closely at this stuff. This is extremely confusing.